src/lib/booleans.js

```diff
@@ -0,0 +1,121 @@
+function negate(a) {
+  if (a === true) {
+    return false;
+  }
+  return true;
+}
+
+function both(a, b) {
+  if (a && b === true) {
+    return true;
+  }
+  return false;
+}
+
+function either(a, b) {
+  if (a || b === true) {
+    return true;
+  }
+  return false;
+}
+
+function none(a, b) {
+  if (!a && !b) {
+    return true;
+  }
+  return false;
+}
+
+function one(a, b) {
+  if ((a && !b) || (!a && b)) {
+    return true;
+  }
+  return false;
```

Comment:
This could be simplified as above by just returning the result of the expression:
```
  return (a && b === false) || (b && a === false);
```